containers/Ideas/ExplorePage.js

```diff
@@ -1,18 +1,30 @@
 import React, { useState, useEffect } from 'react'
 import { View, FlatList } from 'react-native'
 import API from '../../BaseApi'
+import AsyncStorage from '@react-native-async-storage/async-storage'
 import { ExploreListItem } from './ExploreListItem'
 import { styles } from './ExplorePage.styles'
 import { ButtonSignOut } from '../../components/ButtonSignOut'
 import { TextSourceSans } from '../../components/TextSourceSans'
+import {useAuthorization} from '../../containers/Auth/AuthProvider.js'
 
 export const ExplorePage = (props) => {
   const [projects, setProjects] = useState([])
+  const {signOut} = useAuthorization()
 
   const fetchProjects = () => {
-    API.getProjects()
-      .then(response => response && setProjects(response))
-      .catch(error => console.warn(error))
+    AsyncStorage.getItem('authToken')
```

Comment:
more of a general question (to all): if all requests need auth, do we really need to write this in every file? or just move the token fetching into the API methods ?

Reply:
yes i think it grew overtime and now it would make sense to put it into the BaseApi

Reply:
Yeah, probably we should do that.. I think it just evolved like this.. we still need to handle the login then differently though. Maybe an issue, so we can do that in another PR?